libbpfgo.h

```diff
@@ -11,10 +11,8 @@
 #include <bpf/bpf.h>
 #include <bpf/libbpf.h>
 
-int libbpf_print_fn(enum libbpf_print_level level,
```

Comment:
I believe clang-format-12 and clang-format-13 have different opinions about this format. We had to set "clang-format-12" as the default in tracee (and then move to 13, 14, ...). Will check that.